server/export.go

```diff
@@ -106,7 +108,21 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 
 			cmd.SetOut(cmd.OutOrStdout())
 			cmd.SetErr(cmd.OutOrStderr())
-			cmd.Println(string(sdk.MustSortJSON(encoded)))
+			out := sdk.MustSortJSON(encoded)
+
+			if outputDocument == "" {
+				cmd.Println(string(out))
+				return nil
+			}
+
+			var exportedGenDoc tmtypes.GenesisDoc
+			if err = tmjson.Unmarshal(out, &exportedGenDoc); err != nil {
+				return err
+			}
+			if err = exportedGenDoc.SaveAs(outputDocument); err != nil {
```

Comment:
```suggestion
			if err = exportedGenDoc.SaveAs(string(out)); err != nil {
```

nit, we don't need to unmarshal before.